src/foam/nanos/auth/AuthenticatedUserDAO.java

```diff
@@ -65,6 +74,23 @@ public FObject put_(X x, FObject obj) {
       toPut.setSpid(user.getSpid());
     }
 
+    User result = (User) super.find_(x, toPut.getId());
+
+    if (
+      result != null &&
+      ! SafetyUtil.equals(result.getGroup(), toPut.getGroup())
+    ) {
+      boolean hasGroupUpdatePermission = auth.check(x, "group.update." + result.getGroup());
```

Comment:
@kgrgreer Are we sure "group.update.whatever" is the best permission for this purpose? I'd expect that to let me edit a specific group, but not necessarily assign users to it.